toolchains/regenerate.sh

```diff
@@ -38,7 +38,7 @@ esac
 
 # Bazel query is the right command so bazel won't fail itself.
 # Keep bazel versions here at most two: current main version, next version
-for BAZEL_VERSION in "3.7.2" "4.0.0"; do
+for BAZEL_VERSION in "3.7.2" "4.0.0" "4.1.0"; do
```

Comment:
if we're going to skip 4.0.0 then we don't have to leave it here.

Reply:
Removed.